libsolidity/inlineasm/AsmPrinter.cpp

```diff
@@ -116,6 +116,17 @@ string AsmPrinter::operator()(assembly::VariableDeclaration const& _variableDecl
 	return "let " + _variableDeclaration.variable.name + appendTypeName(_variableDeclaration.variable.type) + " := " + boost::apply_visitor(*this, *_variableDeclaration.value);
 }
 
+string AsmPrinter::operator()(assembly::ForLoop const& _forLoop)
+{
+	string out = "for ";
+	out += boost::apply_visitor(_forLoop.preCondition);
```

Comment:
I think this will introduce newlines for the braces. Is there an easy way to avoid that?